Redmine_task_1.rb

```diff
@@ -0,0 +1,213 @@
+require 'selenium-webdriver'
+require 'test-unit'
+
+ class TestRedmine < Test::Unit::TestCase
+ 	
+ 	def setup
+ 		@driver = Selenium::WebDriver.for :firefox
+ 		@wait = Selenium::WebDriver::Wait.new(:timeout => 20)
+ 		@driver.navigate.to 'http://demo.redmine.org/'
+ 		
+ 		@login1 = ('alogin1' + rand(99999).to_s)
+		@pass1 = (9999 + rand(99999))
+		@name1 = ('aname1' + rand(99999).to_s)
+		@lastname1 = ('lastname1' + rand(99999).to_s)
+		@email1 = (@login1 + '@247e.com')
+
+		@new_password = @pass1 + rand(99999)
+
+		@login2 = ('blogin2' + rand(99999).to_s)
+		@pass2 = (9999 + rand(99999))
+		@name2 = ('bname2' + rand(99999).to_s)
+		@lastname2 = ('lastname2' + rand(99999).to_s)
+		@email2 = (@login2 + '@247e.com')
+
+		@project_name = ('project' + rand(99999).to_s)
+		@project_name_version1 = @project_name.reverse
+ 	end
+
+ 	def register_user(login, password, name, lastname, email)
```

Comment:
too much arguments
please think about ways to reduce them
for example some data can be generated just in this method since it's been used only here